src/components/Form/Form.md

```diff
@@ -1,327 +1,184 @@
 ```jsx
-import validateTruthy from '../../validators/validateTruthy'
-import validateMinMaxFactory from '../../validators/validateMinMax'
-import { TitleLarge, TextInput, Spacer, Button, InfoMessage, ZipInput } from '../index'
-let count = 0
-
-const validateCustom = (x) => {
-  console.log("validateCustom got called...")
-  return !!x ? '' : 'Validate custom error'
-}
-
-// This will only be called by the form engine after a field is
-// validated and no errors were found (empty error messages string)
-// Note that it's the consumer's responsibility to keep a table
-// lookup or similar to prevent duplicate tracking of valid fields, etc.
-const analyticsCustomEvent = (fieldName, fieldValue) => {
-  console.log("analyticsCustomEvent got called--field name: ", fieldName, "fieldValue: ", fieldValue, "--means no validation errors, so we'd likely call something like sendAnalyticsEvent(fieldName, fieldValue)")
-}
-
-;<Form
-  config={{
-    formName: 'Styleguidist example form',
-    autocompleteOff: true,
-    formId: '1',
+import { useState } from 'react'
+import validateExists from '../../validators/validateExists'
+import {
+  TitleLarge,
+  TitleSmall,
+  TextInput,
+  Spacer,
+  Button,
+  InfoMessage,
+  ZipInput,
+} from '../index'
+
+const questionGroups = {
```

Comment:
I restructured this so we could key off the question and dynamically create an array from that. I  also added `fields` as that would be closer to real use and allow for more then one field etc.